### PR TITLE
update packages

### PR DESCRIPTION
**Pull request by mgdemel**
Including one breaking change: install axios@1.6.6
Ran dev locally, and nothing broke.

### CODE REVIEW

File: package.json
Comment (joonasnuutinen) [inline package.json:1]: The `"engines"` property sets `"node": "16.x"`. The LTS "recommended for most users" is currently 20. When running `npm i`, I get the following warning:

```bash
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'impact-helper-api@2.2.0',
npm WARN EBADENGINE   required: { node: '16.x' },
npm WARN EBADENGINE   current: { node: 'v20.11.0', npm: '10.2.4' }
npm WARN EBADENGINE }
```

Should we either update the engine to include 20 or omit it altogether?

**Review by joonasnuutinen — COMMENTED** [review]
Seems to work!

File: package.json
Comment (mgdemel) [reply package.json:1]: I'll take a look at the pros and cons

Comment (mgdemel) [reply package.json:1]: I can see that this was set to 16.x in a commit that configured the Heroku deployment. We should check if updating would impact that somehow.

Comment (joonasnuutinen) [reply package.json:1]: Ah, good point! Then let's keep it and just update it to `20.x`.

File: package-lock.json
Comment (joonasnuutinen) [inline package-lock.json:1]: Run `npm i` once again to update the `package-lock` after the Node version change.

**Review by joonasnuutinen — CHANGES_REQUESTED** [review]
Great job! Just one minor thing and this is ready.

File: package-lock.json
Comment (mgdemel) [reply package-lock.json:1]: Done! :)

**Review by joonasnuutinen — APPROVED** [review]
🚀

**Pull request merged**